src/server/protocol.ts

```diff
@@ -868,6 +866,14 @@ namespace ts.server.protocol {
         format?: "original" | "2020"
     }
 
+    /** The response for a EncodedSemanticClassificationsRequest */
+    export interface EncodedSemanticClassificationsResponse extends Response {
+        body?: {
+            endOfLineState: EndOfLineState;
+            spans: number[];

```

Comment:
Dont we need specific enum for  classifications returned otherwise vscode would need separate list?

Reply:
No, I don't think so - the data looks like:

>          * @returns a number array encoded as triples of [start, length, ClassificationType, ...].